src/main/java/client/ClientTemp.java

```diff
@@ -68,11 +68,12 @@ public static void main(String[] args) throws Exception {
         Thread clientThread = new Thread(clientTemp);
         clientThread.start();
         ClientHandler clientHandler = clientTemp.getHandler();
+        Session session = clientTemp.session;
 
         while (!Thread.currentThread().isInterrupted()) {
             String message = scanner.nextLine();
             try {
-                clientHandler.write(message);
+                clientHandler.write(message, session);
```

Comment:
Maybe we don't need the handler to send a message? We have session, user name and message as a string. We have everything we need to create the `Message` and send it